extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/elb.go

```diff
@@ -177,6 +177,11 @@ func convertTextToNLBAccessLogRecord(fields []string) (NLBAccessLogRecord, error
 		return record, fmt.Errorf("could not convert client port to integer: %w", err)
 	}
 
+	record.DestinationIP = strings.Split(fields[6], ":")[0]
+	if record.DestinationPort, err = safeConvertStrToInt(strings.Split(fields[6], ":")[1]); err != nil {
+		return record, fmt.Errorf("could not convert destination port to integer: %w", err)
+	}
```

Comment:
Can that field have IPv6 addresses? Should we use https://pkg.go.dev/net#SplitHostPort?